utils/make_release_notes.sh

```diff
@@ -76,6 +78,15 @@ CYCAMORETXT="$NCOMMITS commits resulting in $SUMMARY"
 CYCAMORECONTRIB=`git log --format="%aN" $PREV...$VERSION | sort -u`
 cd $HERE
 
+# cymetric summary
+cd $CYMETRIC
+[[ `git tag` =~ $PREV ]] && CYMETRIC_PREV=$PREV || CYMETRIC_PREV=`git log --pretty=format:%H|tail -1`
+NCOMMITS=`git rev-list CYMETRIC_PREV...$VERSION --count | tail -n1`
```

Comment:
before you run type `export CYCLUS_DIR=/path/to/cyclus`, etc.

On Thu, May 7, 2015 at 3:22 PM, opotowsky <notifications@github.com> wrote:

> In utils/make_release_notes.sh
> <https://github.com/gidden/ciclus/pull/2#discussion_r29891954>:
>
> > @@ -76,6 +78,15 @@ CYCAMORETXT="$NCOMMITS commits resulting in $SUMMARY"
> >  CYCAMORECONTRIB=`git log --format="%aN" $PREV...$VERSION | sort -u`
> >  cd $HERE
> >
> > +# cymetric summary
> > +cd $CYMETRIC
> > +[[ `git tag` =~ $PREV ]] && CYMETRIC_PREV=$PREV || CYMETRIC_PREV=`git log --pretty=format:%H|tail -1`
> > +NCOMMITS=`git rev-list CYMETRIC_PREV...$VERSION --count | tail -n1`
>
> got it! thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gidden/ciclus/pull/2/files#r29891954>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192